src/main/java/sx/blah/discord/handle/impl/obj/User.java

```diff
@@ -276,6 +276,13 @@ public void setDiscriminator(String discriminator) {
 	}
 
 	@Override
+	public EnumSet<Permissions> getPermissionsForGuild(IGuild guild){
```

Comment:
You should update Channel#getModifiedPermissions to use this method for consistency.